tests/template_tests/syntax_tests/test_if.py

```diff
@@ -537,3 +537,14 @@ def test_if_is_match(self):
     def test_if_is_no_match(self):
         output = self.engine.render_to_string('template', {'foo': 1})
         self.assertEqual(output, 'no')
+
+    @setup({'template': '{% if foo is not None %}yes{% else %}no{% endif %}'})
+    def test_if_is_not_match(self):
+        # It's important not to use foo=True, because True is (not None)
```

Comment:
"For this to act as a regression test, it's important..."
(chop comma before "because")